homeassistant/components/knx/storage/entity_store_schema.py

```diff
@@ -106,6 +113,18 @@ def optional_ga_schema(key: str, ga_selector: GASelector) -> VolDictType:
     }
 )
 
+SENSOR_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_ENTITY): BASE_ENTITY_SCHEMA,
+        vol.Required(DOMAIN): {
+            vol.Required(CONF_GA_SENSOR): GASelector(state_required=True),
+            vol.Required(CONF_VALUE_TYPE): str,
```

Comment:
```suggestion
            vol.Required(CONF_GA_SENSOR): GASelector(
                write=False,
                state_required=True,
                dpt=[],
            ),
```
The interesting thing for UI configurable sensor entities will be how to pass and validate available `dpt` from `xknx.dpt` up to the UI and back.
`value_type` can be omitted then.